ReleaseTooling/Sources/ZipBuilder/ZipBuilder.swift

```diff
@@ -452,28 +453,76 @@ struct ZipBuilder {
                                        rootZipDir: zipDir,
                                        builtFrameworks: frameworksToAssemble,
                                        podsToIgnore: analyticsPods)
-
         // Update the README.
         readmeDeps += dependencyString(for: pod.key, in: productDir, frameworks: podFrameworks)
-
-        // Special case for Crashlytics:
-        // Copy additional tools to avoid users from downloading another artifact to upload symbols.
-        let crashlyticsPodName = "FirebaseCrashlytics"
-        if pod.key == crashlyticsPodName {
-          for file in ["upload-symbols", "run"] {
-            let source = pod.value.installedLocation.appendingPathComponent(file)
-
-            let target = zipDir.appendingPathComponent(crashlyticsPodName)
-              .appendingPathComponent(file)
-            do {
-              try FileManager.default.copyItem(at: source, to: target)
-            } catch {
-              fatalError("Error copying Crashlytics tools from \(source) to \(target): \(error)")
+      } catch {
+        fatalError("Could not copy frameworks from \(pod) into the zip file: \(error)")
+      }
+      do {
+        // Update Resources: For the zip distribution, they get pulled from the xcframework to the
+        // top-level product directory. For the Carthage distribution, they propagate to each
+        // individual framework.
+        // TODO: Investigate changing the zip distro to also have Resources in the .frameworks to
+        // enable different platform Resources.
+        let productPath = zipDir.appendingPathComponent(pod.key)
+        let contents = try fileManager.contentsOfDirectory(atPath: productPath.path)
+        for fileOrFolder in contents {
+          let xcPath = productPath.appendingPathComponent(fileOrFolder)
+          let xcResourceDir = xcPath.appendingPathComponent("Resources")
+
+          // Ignore anything that not an xcframework with Resources
+          guard fileManager.isDirectory(at: xcPath),
+            xcPath.lastPathComponent.hasSuffix("xcframework"),
+            fileManager.directoryExists(at: xcResourceDir)
+          else { continue }
+
+          if packageKind == "Firebase" {
+            // Move all the bundles in the frameworks out to a common "Resources" directory to
+            // match the existing Zip structure.
+            let resourcesDir = productPath.appendingPathComponent("Resources")
+            try fileManager.moveItem(at: xcResourceDir, to: resourcesDir)
```

Comment:
moveItem instead of `ResourcesManager.moveAllBundles` since the resources were already processed when the xcframework was created.